email/track-editing-allotment.html

```diff
@@ -26,22 +26,22 @@
     <tbody valign="top">
       {% for task in params.tasks %}
       <tr>
-        <td>{{ task.outputFileLink }}</td>
+        <td>{{ task.fileName }}</td>
         <td>
-          {% for chunk in task.chunks %}
-          {{ chunk.fileName }};&nbsp;{{ chunk.beginning }}-{{ chunk.ending
+          {% for chunk in task.task.chunks %}
+          <a href="{{ params.fileLinkBaseUrl }}/{{ task.listName }}/{{ chunk.fileName }}">{{ chunk.fileName }}</a>&nbsp;{{ chunk.beginning }}&dash;{{ chunk.ending
```

Comment:
1. I would prefer if we construct the links in the function code and inject them into chunks objects rather than putting this logic in the template. So here you should refer them as `chunk.sourceFileLink`.
1. Hypothetically one track editing task can consist of the chunks from different lists. It may be rare or even not be the case at all, only practice will show. But to be safe use each chunk file name to get its list and construct the `sourceFileLink`.
1. `&dash;` is the same as hyphen. You should use `&ndash;` here. See https://dev.w3.org/html5/html-author/charref.